serverlessworkflow/modules/ROOT/pages/cloud/enhanced-service-discovery.adoc

```diff
@@ -0,0 +1,128 @@
+= Enhanced Service Discovery
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
+// links
+:quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
+
+
+The Enhanced Service Discovery allows you to have a static URI that will define the Kubernetes resource that you use
+to perform http requests, the following sketch demonstrate how the SWUrl is composed:
+
+[source,shell]
+----
+kubernetes:<group>/<version>/<kind>/<namespace>/<name>?<attributeName>=<attributeValue>
+\________/ \_____/ \_______/ \____/ \_________/ \____/ \______________________________/
+  scheme    group   version   kind   namespace   name   additional resource attributes
+            \____________________/                       \__________________________/
+                     GVK                                  Supported value: port_name
+----
+
+On the scheme, the following values are supported:
+
+* **kubernetes**
+* **openshift**
+* **knative**
+
+As GVK, for now, the following Kubernetes Resources are supported:
+
+* **v1/service**
+* **serving.knative.dev/v1/service**
+* **v1/pod**
+* **apps/v1/deployment**
+* **apps.openshift.io/v1/deploymentconfig**
+* **apps/v1/statefulset**
+* **route.openshift.io/v1/route**
+* **networking.k8s.io/v1/ingress**
+
+[TIP]
+====
+Note that the _GVK_(Group, Version and Kind) and the _Resource Name_ are mandatory and can't be empty, otherwhise, the engine will throw a validation issue and the discovery will not be performed. The **namespace** is not mandatory, however, if empty, the current namespace or context will be used.
+====
```

Comment:
```suggestion
[IMPORTANT]
====
The **GVK** (Group, Version and Kind) and the **Name** values are **mandatory** and can't be empty, otherwise, the engine will throw a validation issue and the discovery will not be performed.  +
The **namespace** is **optional**. In case of an empty value, the current namespace or context will be used.
====
```